buildscripts/manylinux_2_17/build_llvmlite.sh

```diff
@@ -31,6 +31,10 @@ export PATH=/opt/python/$pyver/bin:$PATH
 
 echo "Using python: $(which python)"
 
+if [[ "$PYVER"="cp27-cp27mu" ]]; then
```

Comment:
I don't know what this is, but it looks related to Python 2.7 - since we only support Python 3, I would imagine this can be removed.